io.qt.qtwebengine.BaseApp.json

```diff
@@ -188,6 +188,7 @@
                 }
             ],
             "build-commands": [
+                "ARCH_TRIPLE=$(gcc --print-multiarch); qtver=$(qmake -query QT_VERSION); qtwebver=$(grep -o '5[0-9.]*' ${FLATPAK_DEST}/lib/${ARCH_TRIPLE}/cmake/Qt5WebEngine/Qt5WebEngineConfigVersion.cmake); lesser_ver=$(echo -e \"${qtver}\n${qtwebver}\" | sort -V | head -1); sed -i \"s/$qtwebver \\(\\${_Qt5WebEngine\\)/\1 $lesser_ver/\" ${FLATPAK_DEST}/lib/${ARCH_TRIPLE}/cmake/Qt5WebEngine*/Qt5WebEngine*Config.cmake",
```

Comment:
>`\"s/$qtwebver \\(\\${_Qt5WebEngine\\)/\1 $lesser_ver/\"`

This doesn't look right. You're putting `${_Qt5WebEngine` first, before the version.

Reply:
Right, I think it's fixed now.